unlonely-alpha/web/src/apiClient/client.ts

```diff
@@ -29,10 +29,7 @@ const authLink = setContext(async (_, { headers }) => {
 
 function createApolloClient() {
   const httpLink = new HttpLink({
-    uri:
-      process.env.NODE_ENV === "production"
-        ? server
-        : "http://localhost:4000/graphql",
+    uri: "https://sea-lion-app-j3rts.ondigitalocean.app/graphql",
```

Comment:
change this back to 

`process.env.NODE_ENV === "production"
        ? server
        : "http://localhost:4000/graphql",`

Reply:
yes